hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -983,6 +983,12 @@ options are covered in [Testing](./testing.md).
   <description>Select which version of the S3 SDK's List Objects API to use.
   Currently support 2 (default) and 1 (older API).</description>
 </property>
+
+<property>
+  <name>fs.s3a.connection.request.timeout</name>
+  <value>0</value>
+  <description>Request timeout in milliseconds. Any non-positive value disables the timeout.</description>
```

Comment:
Please describe what happens if the timeout is disabled. Is 0 a non-positive value? What will happen if the setting is 0?
Also, please include what will happen if the request will time out in cases like copying several large files.

Reply:
if you pass in a negative number intOption will reject. With a move to getTimeDuration you get to specify millis and hours, but you can't use negative time